src/Transports/MassTransit.KafkaIntegration/Testing/KafkaTestHarnessHostedService.cs

```diff
@@ -79,7 +79,7 @@ async Task CreateTopics(IReadOnlyList<string> topicNames, KafkaTestHarnessOption
                 if (topics.Any())
                 {
                     // Retry failing topics creation
-                    await Task.Delay(10).ConfigureAwait(false);
+                    await Task.Delay(100).ConfigureAwait(false);
```

Comment:
Why wouldn't you just use Retry. with an actual retry policy instead of some arbitrary delay? Also, if the topic already exists the log message in the test is slightly annoying.

Reply:
it is retrying only topics with that `Reason`